src/core/ledger/transaction.types.d.ts

```diff
@@ -5,7 +5,7 @@ type IAddTransactionOnUsPayment = {
   metadata: Record<string, unknown>
   payerUser: UserType
   payeeUser: UserType
-  memoPayer?: string
+  memoFromPayer?: string
```

Comment:
Why nullable? Couldn't we default to empty String `""`?

Reply:
Changed here: https://github.com/GaloyMoney/galoy/pull/415/commits/a3336e2c900468af631065e04b76afcb255cd7e1

Reply:
I undid the above change because it cascaded into changing a number of other functions where the parameter types would need to be redefined and function calls changed.